### PR TITLE
reload json module added

### PR DESCRIPTION
**Pull request by dimacodota**
this PR contains the following: 
 - clear cached version of the json module and reload  it
 - "Enable Native Auto Complete" command
 - "Disable/Enable for Current View" commands

### CODE REVIEW

File: lib/logger.py [hunk 1]
```diff
@@ -11,7 +10,9 @@ def debug(msg, if_development=True):
         info(msg)
 
 def jsonstr(obj):
-    return json.dumps(obj, indent=2)
+	from json import dumps
```
Comment (boaz-codota) [inline lib/logger.py:13]: why import here and not at the start?

File: lib/logger.py [hunk 2]
```diff
@@ -11,7 +10,9 @@ def debug(msg, if_development=True):
...
+	return dumps(obj, indent=2)
+    # return dumps(obj, indent=2)
```
Comment (boaz-codota) [inline lib/logger.py:15]: remove comments

File: lib/tab_nine_process.py [hunk 3]
```diff
@@ -98,6 +95,7 @@ def restart_tabnine_proc(self):
         self.tabnine_proc = TabNineProcess.run_tabnine()
 
     def request(self, req):
+        from json import loads, dumps
```
Comment (boaz-codota) [inline lib/tab_nine_process.py:98]: why here as well? Is there an issue with json library?

File: completions/completions_v2.py [hunk 4]
```diff
@@ -94,13 +76,14 @@ def on_selection_modified(self, view):
         self.on_any_event(view)
     def on_activated(self, view):
         self.on_any_event(view)
-        if view.window():
-            view.window().status_message("TabNine")
+        view.set_status("tabnine-status", "TabNine")
```
Comment (boaz-codota) [inline completions/completions_v2.py:79]: What this does?

File: TabNine.py [hunk 5]
```diff
@@ -1,11 +1,49 @@
-from .tab_nine_process import tabnine_proc
-from .lib.settings  import is_native_auto_complete
 import sublime_plugin
 import sublime
+import sys
+
+if int(sublime.version()) >= 3114:
```
Comment (boaz-codota) [inline TabNine.py:5]: Can we extract this to a function, and give it a name?

**Review by boaz-codota — APPROVED** [review]

File: completions/completions_v2.py (hunk 4 above)
Comment (dimacodota) [reply completions/completions_v2.py:79]: sets the status bar item to "TabNine" 
Screen Shot 2020-10-18 at 12 08 23

File: lib/logger.py (hunk 1 above)
Comment (dimacodota) [reply lib/logger.py:13]: it should be lazily imported

File: lib/tab_nine_process.py (hunk 3 above)
Comment (dimacodota) [reply lib/tab_nine_process.py]: yes, all this PR is related to an issue with Json library, in some cases, another unknown version of it is cashed, and is returned by the import statement. so the solution is to clear it from the cache and reload it

File: lib/logger.py (hunk 2 above)
Comment (dimacodota) [reply lib/logger.py]: done

File: TabNine.py (hunk 5 above)
Comment (dimacodota) [reply TabNine.py]: done

**Pull request merged**